uwsgi/admin.ini

```diff
@@ -11,12 +11,16 @@ check-static = /app/ui/dist
 # Serve index.html when / is requested
 static-index = index.html
 
-# Always set REMOTE_USER
-# TODO: Where to put this when this config is shared by production?
-route = ^/.*$ addvar:REMOTE_USER=balrogadmin
-
-# Skip further routing for requests to /api
-route-if = startswith:${REQUEST_URI};/api continue:
+# Because the Angular app and the WSGI app are both hosted at /, we need to
+# set-up some fancy routing to cope. Specifically, we need to take any URIs
+# that should be for the admin app, and direct them there. These are:
+# Anything coming in to /api, which also needs to be stripped away before
```

Comment:
So long as we're hosting the UI and the wsgi app in the same place I think there's risk of pain, I think. Admittedly, serving them both out of / instead of putting the wsgi app in /api is worse, but I couldn't find a way to keep it in /api and make the Dockerflow endpoints work :(. Ultimately, I think a better solution is to host them on different domains, eg: aus-api.mozilla.org and aus-admin.mozilla.org.